### PR TITLE
refactor: use branches for backoffice interfaces

### PR DESCRIPTION
**Pull request by caspiano**
Default to `backoffice-alpha` in development, `backoffice-release` in production

### CODE REVIEW

File: src/tasks.cr [hunk 1]
```diff
@@ -5,4 +5,10 @@ module PlaceOS::Tasks
   include Database
   include Entities
   include Initialization
+
+  PROD = (ENV["ENV"]? || ENV["SG_ENV"]?) == "production"
```
Comment (KimBurgess) [inline src/tasks.cr:9]: I may be mistaken, but think this result in the compile time values for these env vars, whereas it looks like runtime is desired.

Comment (caspiano) [reply src/tasks.cr:9]:
```crystal
CONSTANT = ENV["FOO"]? || "runtime"
puts CONSTANT
```

```bash
$ crystal build test.cr
$ ./test
runtime
$ FOO=hello ./test
hello
```

Comment (KimBurgess) [reply src/tasks.cr:9]: Oh nice! My mental model for that was well and truly broken.

Comment (caspiano) [reply src/tasks.cr:9]: [This is the compile time env lookup](https://crystal-lang.org/api/0.35.1/Crystal/Macros.html#env(name):StringLiteral|NilLiteral-instance-method)

**Pull request merged**